src/Lesson5/Mang6.java

```diff
@@ -11,20 +11,20 @@ public static void main(String[] args) {
         A[4] = 5;
         A[5] = 19;
         A[6] = 11;
-        int n = 7; // Số phần tử có trong mảng
-        System.out.println("Trước khí xóa: ");
+        int n = 10 ; // Số phần tử có trong mảng
+        System.out.println("Trước khi delete: ");
         for (int i = 0; i < n; i++)
             System.out.print(A[i] + "   ");
-        int x = 7;
+        int x = 7 ;
         n = Delete(A, n, x);
-        System.out.println("\nSau khi xóa : ");
+        System.out.println("\nSau khi delete : ");
         for (int i = 0; i < n; i++)
             System.out.print(A[i] + "   ");
     }
     public static int Delete(int[] A, int n, int x)
     {
         // Xoa phan tu
-        for (int i = 1; i < n; i++) {
+        for (int i = 0  ; i < n; i++) {
```

Comment:
ở đây e truyền biến int x vào ý là xóa phần tử tại vị trí x. Vậy mình sẽ phải dịch tất cả các phần tử sau x sang bên trái. 
nên trong vòng for phải là: for (int i = x; i < n; i++)...